include/Config.h

```diff
@@ -199,16 +216,26 @@ class Config {
       return errors;
     }
 
-    std::ifstream f(bitcodeFileList.c_str());
+    std::ifstream bitcodeFile(bitcodeFileList.c_str());
 
-    if (f.good() == false) {
+    if (bitcodeFile.good() == false) {
       std::stringstream error;
 
       error << "bitcode_file_list parameter points to a non-existing file: "
             << bitcodeFileList;
 
       errors.push_back(error.str());
-      return errors;
+    }
+    
+    std::ifstream dynamicLibraryFile(dynamicLibraryFileList.c_str());
+    
+    if (dynamicLibraryFile.good() == false && dynamicLibraryFileList.empty() == false) {
```

Comment:
Given `dynamicLibraryFile` is not a required but optional parameter, please do it like:

```cpp
if (dynamicLibraryFileList.empty() == false) {
  std::ifstream dynamicLibraryFile(dynamicLibraryFileList.c_str()); 
  if (dynamicLibraryFile.good() == false) {
    ...
  }
}
```